app/src/main/res/values/styles.xml

```diff
@@ -11,9 +11,10 @@
     <style name="AppTheme.NoActionBar">
         <item name="windowActionBar">false</item>
         <item name="windowNoTitle">true</item>
+        <item name="colorPrimaryDark">@color/white</item>
```

Comment:
Can you make the white colour darker

Reply:
Do you mean grey ?

Reply:
Kind of. see the eventbrite app,the notifications of the user should be visible

Reply:
I think then the let the colour be kept as default colorPrimaryDark (blue)